pytorch_pfn_extras/training/extensions/variable_statistics_plot.py

```diff
@@ -260,7 +260,7 @@ def available():
         return _available
 
     def __call__(self, manager):
-        if self.available():
+        if _available:
```

Comment:
Availability check is not performed? `_check_available()`

Reply:
I overlooked that `VariableStaticsPlot` didn't call it in `__init__`. Fixed!

Reply:
Thanks!!! 🙇‍♂️